src/js/utils/Units.js

```diff
@@ -55,9 +54,7 @@ const Units = {
       return amount;
     }
 
-    options = _.extend({
-      decimalPlaces: 2
-    }, options);
+    options = Object.assign({decimalPlaces: 2}, options || {});
```

Comment:
Yeah, sorry. If we are only checking for `undefined` (and not `null` or other falsy values) when defaulting `options` to `{}` we could use default assign in the arguments of the function:
```js
contractNumber: function (amount, options = {}) {
```

This will only assign `options` if the argument is `undefined`, not if it `null` or other falsy values.